plugin-server/src/worker/ingestion/event-pipeline/runner.ts

```diff
@@ -87,14 +87,14 @@ export class EventPipelineRunner {
         this.hub.statsd?.increment('kafka_queue.event_pipeline.start', { pipeline: 'event' })
 
         try {
-            if (this.isEventBlacklisted(event)) {
+            if (this.isEventDisallowed(event)) {
                 eventDroppedCounter
                     .labels({
                         event_type: 'analytics',
-                        drop_cause: 'blacklisted',
+                        drop_cause: 'disallowed',
```

Comment:
Could this impact analytics that are relied on?

Reply:
Great question.

@tiina303 Git blame tells me you were the last around these parts of the code. Do you know whether there are any implications for changing this?

Reply:
ooh indeed great catch, it is changing the metric label, so yes filtering for it in `grafana-prod-us` would break / need updating

Reply:
All good with this one. For any metrics I'd just search in the charts repo, where we have dashboards/alerts https://github.com/search?q=repo%3APostHog%2Fcharts%20blacklisted&type=code

Reply:
Sorry, I'm rather new with Grafana and I'm not too sure what I'm looking for in there. Considering the search in the charts repo returns nothing for me, are we all good, or are those two things not connected?